### PR TITLE
[DOCS] OBSDOCS-475 - Logging 5.7.6 Release Notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
**Change type:** Doc update; Logging Z-Stream Release Notes - 5.7.6
**Doc JIRA:** https://issues.redhat.com/browse/OBSDOCS-475

**Fix Version:** 4.11+

**Doc Preview:** https://64615--docspreview.netlify.app/openshift-enterprise/latest/logging/v5_7/logging-5-7-release-notes#cluster-logging-release-notes-5-7-6_logging-5.7-release-notes

**SME Review:** @xperimental 
**QE Review:** @kabirbhartiRH 
**Peer Review:** @libander & @briandooley

### CODE REVIEW

**Comment by shreyasiddhartha** [comment]
@kabirbhartiRH I think we only document "public" bugs available in the errata and not the "private" ones.
@libander any comments on this?

File: modules/logging-rn-5.7.6.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+This release includes link:https://access.redhat.com/errata/RHSA-2023:4933[OpenShift Logging Bug Fix Release 5.7.6].
+
+[id="openshift-logging-5-7-6-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, the collector relied on the default configuration settings for reading the container log lines. As a result, the collector did not read the rotated files efficiently. With this update, there is increase in the number of bytes read, which allows the collector to efficiently process rotated files. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])
+
+* Before this update, when users pasted a URL with predefined filters, some filters did not reflect. With this update, the UI reflects all the filters in the URL. (link:https://issues.redhat.com/browse/LOG-4459[LOG-4459])
+
+* Before this update, the Vector forwarding to Loki using custom labels was not usable when switching from Vector to Loki as a collector. As a result, the collector caused an error. With this update, the Vector configuration sanitizes labels in the same way as the Fluentd Loki plugin to ensure the collector starts and correctly processes labels. (link:https://issues.redhat.com/browse/LOG-4460[LOG-4460])
```
Comment (libander) [inline modules/logging-rn-5.7.6.adoc:15]: * Before this update, forwarding to Loki using custom labels generated an error when switching from Fluentd to Vector.  With this update, the Vector configuration sanitizes labels in the same way as Fluentd to ensure the collector starts and correctly processes labels. (link:https://issues.redhat.com/browse/LOG-4460[LOG-4460])

**Comment by kabirbhartiRH** [comment]
> @kabirbhartiRH I think we only document "public" bugs available in the errata and not the "private" ones. @libander any comments on this?

I went back and checked for Logging 5.7.4 https://errata.devel.redhat.com/advisory/117246
It seems we have included such bugs in Docs. AFAIK, we don't include NEC restricted bugs in docs but may include those which are RH employee/contractor restricted.

File: modules/logging-rn-5.7.6.adoc (hunk 1 above)
Comment (shreyasiddhartha) [reply modules/logging-rn-5.7.6.adoc:15]: Fixed this. Thanks @libander. 👍

**Comment by shreyasiddhartha** [comment]
> > @kabirbhartiRH I think we only document "public" bugs available in the errata and not the "private" ones. @libander any comments on this?
> 
> I went back and checked for Logging 5.7.4 https://errata.devel.redhat.com/advisory/117246 It seems we have included such bugs in Docs. AFAIK, we don't include NEC restricted bugs in docs but may include those which are RH employee/contractor restricted.
> 
> for e.g. #62992, [LOG-3381](https://issues.redhat.com//browse/LOG-3381) is included but other restricted bugs are not (NEC one's)

@kabirbhartiRH Thanks for clarifying. I have included the bugs. PTAL. :)

**Comment by briandooley** [comment]
Release notes text in [LOG-4456](https://issues.redhat.com/browse/LOG-4456)was found to be out of date during QE review. It has been changed and we are currently altering the PR with the new information.

File: modules/logging-rn-5.7.6.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+* Before this update, the Observability Loki console search field did not accept special characters that should escape. With this update, it allows special characters to escape by using the content filter. (link:https://issues.redhat.com/browse/LOG-4456[LOG-4456])
+
+* Before this update, the following warning message appeared while sending logs to Splunk: `Timestamp was not found.` With this update, the change overrides the name of the log field used to retrieve the Timestamp and sends it to Splunk without warning. (link:https://issues.redhat.com/browse/LOG-4413[LOG-4413])
+
+* Before this update, The CPU and the memory usage of Vector was increasing over time. With this update, the Vector configuration now contains the `expire_metrics_secs=60` setting to limit the lifetime of the metrics and cap the associated CPU usage and memory footprint. (link:https://issues.redhat.com/browse/LOG-4171[LOG-4171])
```
Comment (xperimental) [inline modules/logging-rn-5.7.6.adoc:21]:
```suggestion
* Before this update, the CPU and memory usage of Vector was increasing over time. With this update, the Vector configuration now contains the `expire_metrics_secs=60` setting to limit the lifetime of the metrics and cap the associated CPU usage and memory footprint. (link:https://issues.redhat.com/browse/LOG-4171[LOG-4171])
```

File: modules/logging-rn-5.7.6.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-7-6_{context}"]
+= Logging 5.7.6
+This release includes link:https://access.redhat.com/errata/RHSA-2023:4933[OpenShift Logging Bug Fix Release 5.7.6].
+
+[id="openshift-logging-5-7-6-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, the collector relied on the default configuration settings for reading the container log lines. As a result, the collector did not read the rotated files efficiently. With this update, there is increase in the number of bytes read, which allows the collector to efficiently process rotated files. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])
+
+* Before this update, when users pasted a URL with predefined filters, some filters did not reflect. With this update, the UI reflects all the filters in the URL. (link:https://issues.redhat.com/browse/LOG-4459[LOG-4459])
```
Comment (xperimental) [inline modules/logging-rn-5.7.6.adoc:13]:
```suggestion
* Before this update, when users pasted an URL with predefined filters, some filters did not reflect. With this update, the UI reflects all the filters in the URL. (link:https://issues.redhat.com/browse/LOG-4459[LOG-4459])
```

File: modules/logging-rn-5.7.6.adoc [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+* Before this update, forwarding to Loki using custom labels generated an error when switching from Fluentd to Vector. With this update, the Vector configuration sanitizes labels in the same way as Fluentd to ensure the collector starts and correctly processes labels. (link:https://issues.redhat.com/browse/LOG-4460[LOG-4460])
+
+* Before this update, the Observability Loki console search field did not accept special characters that should escape. With this update, it allows special characters to escape by using the content filter. (link:https://issues.redhat.com/browse/LOG-4456[LOG-4456])
```
Comment (xperimental) [inline modules/logging-rn-5.7.6.adoc:17]: Not so sure about this one. The console is reachable using a "Logs" link in the UI, but it _is_ only for querying our default instance of Loki, if that's available.

```suggestion
* Before this update, the Observability Logs console search field did not accept special characters that it should escape. With this update, it is escaping special characters properly in the query. (link:https://issues.redhat.com/browse/LOG-4456[LOG-4456])
```

**Review by xperimental — COMMENTED** [review]
Only some small changes.

File: modules/logging-rn-5.7.6.adoc (hunk 2 above)
Comment (shreyasiddhartha) [reply modules/logging-rn-5.7.6.adoc]: Made this change. Thanks!

File: modules/logging-rn-5.7.6.adoc (hunk 3 above)
Comment (shreyasiddhartha) [reply modules/logging-rn-5.7.6.adoc]: Made this change. Thanks!

**Comment by shreyasiddhartha** [comment]
@xperimental I have made the changes suggested by you. PTAL and give LGTM if it's okay. Thanks! :)

**Review by xperimental — APPROVED** [review]

File: modules/logging-rn-5.7.6.adoc [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+// cluster-logging-release-notes.adoc
+// logging-5-7-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-7-6_{context}"]
+= Logging 5.7.6
+This release includes link:https://access.redhat.com/errata/RHSA-2023:4933[OpenShift Logging Bug Fix Release 5.7.6].
+
+[id="openshift-logging-5-7-6-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, the collector relied on the default configuration settings for reading the container log lines. As a result, the collector did not read the rotated files efficiently. With this update, there is increase in the number of bytes read, which allows the collector to efficiently process rotated files. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])
```
Comment (briandooley) [inline modules/logging-rn-5.7.6.adoc:11]:
```suggestion
* Before this update, the collector relied on the default configuration settings for reading the container log lines. As a result, the collector did not read the rotated files efficiently. With this update, there is an increase in the number of bytes read, which allows the collector to efficiently process rotated files. (link:https://issues.redhat.com/browse/LOG-4501[LOG-4501])
```

File: modules/logging-rn-5.7.6.adoc [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+* Before this update, when users pasted an URL with predefined filters, some filters did not reflect. With this update, the UI reflects all the filters in the URL. (link:https://issues.redhat.com/browse/LOG-4459[LOG-4459])
```
Comment (briandooley) [inline modules/logging-rn-5.7.6.adoc:13]:
```suggestion
* Before this update, when users pasted a URL with predefined filters, some filters did not reflect. With this update, the UI reflects all the filters in the URL. (link:https://issues.redhat.com/browse/LOG-4459[LOG-4459])
```

File: modules/logging-rn-5.7.6.adoc [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+* Before this update, the following warning message appeared while sending logs to Splunk: `Timestamp was not found.` With this update, the change overrides the name of the log field used to retrieve the Timestamp and sends it to Splunk without warning. (link:https://issues.redhat.com/browse/LOG-4413[LOG-4413])
+
+* Before this update, the CPU and memory usage of Vector was increasing over time. With this update, the Vector configuration now contains the `expire_metrics_secs=60` setting to limit the lifetime of the metrics and cap the associated CPU usage and memory footprint. (link:https://issues.redhat.com/browse/LOG-4171[LOG-4171])
+
+* Before this update, the LokiStack gateway cached authorized requests very broadly. As a result, this caused wrong authorization results. With this update, LokiStack gateway caches on a more fine-grained basis which resolves this issue. (link:https://issues.redhat.com/browse/LOG-4393[LOG-4393])
+
+* Before this update, the Fluentd runtime image included builder tools which were unnecessary at runtime. With this update, the builder tools are removed resolving the issue. (link:https://issues.redhat.com/browse/LOG-4467[LOG-4467])
```
Comment (briandooley) [inline modules/logging-rn-5.7.6.adoc:25]:
```suggestion
* Before this update, the Fluentd runtime image included builder tools which were unnecessary at runtime. With this update, the builder tools are removed, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4467[LOG-4467])
```

**Review by briandooley — COMMENTED** [review]
Some minors grammar comments.

**Pull request merged**